lua/cmp_tmux/tmux.lua

```diff
@@ -51,7 +51,7 @@ end
 function Tmux.create_pane_data_job(self, pane, on_data, on_exit)
     local cmd = { 'tmux', 'capture-pane', '-p', '-t', pane }
     if self.config.capture_history then
-        cmd:append('-S -')
+        cmd = vim.tbl_extend('force', cmd, {'-S', '-'})
```

Comment:
Running `vim.tbl_extend` probably adds some overhead compared to a `table.insert` 🤔